core/vm/evmc.go

```diff
@@ -311,6 +311,13 @@ func (evm *EVMC) Run(contract *Contract, input []byte, readOnly bool) (ret []byt
 		panic(fmt.Sprintf("EVMC VM internal error: %s", evmcError.Error()))
 	}
 
+	if len(output) > params.MaxCodeSize {
```

Comment:
This should checked only for CREATE outputs, right?

Reply:
yes. Do you mean that I should add a check for `kind == kind = evmc.Create` ?